browser/ui/views/playlist/playlist_action_dialogs.cc

```diff
@@ -251,6 +253,32 @@ void ClosePanel(content::WebContents* contents) {
   }
 }
 
+namespace {
+class PlaylistLabelButton : public views::LabelButton {
+  METADATA_HEADER(PlaylistLabelButton, views::LabelButton)
+ public:
+  PlaylistLabelButton(views::Button::PressedCallback callback = {},
+                      const std::u16string& text = std::u16string())
+      : LabelButton(std::move(callback), text) {
+    SetEnabledTextColorIds(kColorBravePlaylistTextInteractive);
+    label()->SetFontList(label()->font_list().Derive(
+        0, gfx::Font::FontStyle::NORMAL, gfx::Font::Weight::NORMAL));
```

Comment:
I think `SEMIBOLD` would have value of 600, which matches our design guide.

```suggestion
        0, gfx::Font::FontStyle::NORMAL, gfx::Font::Weight::SEMIBOLD));
```

Reply:
Done! Also changed the font size to 13px as per you suggestion from [here](https://github.com/brave/brave-browser/issues/40243#issuecomment-2276924862).